### PR TITLE
implement homework 4

### PR DESCRIPTION
**Pull request by marianmagur**

### CODE REVIEW

File: homeworks/magurmarian/hw4/tests.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+parser = argparse.ArgumentParser(description="Parsing letter for mapping")
+parser.add_argument('--letter', required=True, type=str)
+args, sys.argv[1:] = parser.parse_known_args(sys.argv[1:])
+
+
+class Test(Testcase):
+
+    letter = mapping[args.letter]
+
+    @loop(number=mapping[args.letter])
```
Comment (Tempos) [inline homeworks/magurmarian/hw4/tests.py]: @marianmagur same as in hw3, why don't you use "letter" variable here?

Comment (marianmagur) [reply homeworks/magurmarian/hw4/tests.py]: @Tempos  Fixed!

**Review by Tempos — APPROVED** [review]
Sorry for the delay. Looks fine.
@extsoft please look at it yourself.

**Comment by marianmagur** [comment]
@extsoft your turn

**Review by extsoft — APPROVED** [review]
@marianmagur good job!

**Pull request merged**